code/modules/paperwork/carbonpaper.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/paper/carbon
+	name = "sheet of carbon"
+	icon_state = "paper_stack"
+	item_state = "paper"
+	var/copied = FALSE
+	var/iscopy = FALSE
+
+/obj/item/paper/carbon/update_icon_state()
+	if(iscopy)
+		if(info)
+			icon_state = "cpaper_words"
+			return
+		icon_state = "cpaper"
+
+	else if(copied)
+		if(info)
+			icon_state = "paper_words"
+			return
+		icon_state = "paper"
+	else
+		if(info)
+			icon_state = "paper_stack_words"
+			return
+		icon_state = "paper_stack"
```

Comment:
instead of this ridiculous chain like this, do something like
```
if (iscopy)
	icon_state = "cpaper"
else if (copied)
	icon_state = "paper"
...
if (info)
	icon_state = "[icon_state]_words"
```

Reply:
It be done two.